src/core/public/workspace/workspaces_service.ts

```diff
@@ -33,24 +21,15 @@ enum WORKSPACE_ERROR_REASON_MAP {
   WORKSPACE_STALED = 'WORKSPACE_STALED',
 }
 
-/**
- * @public
- */
-export interface WorkspacesSetup extends WorkspaceObservables {
-  registerWorkspaceMenuRender: (render: WorkspaceMenuRenderFn) => void;
-}
-
-export interface WorkspacesStart extends WorkspaceObservables {
-  renderWorkspaceMenu: () => JSX.Element | null;
-}
+export type WorkspacesSetup = WorkspaceObservables;
```

Comment:
Do we still need to export type `WorkspaceObservables`?

Reply:
If `WorkspaceObservables` is no longer needed, we can reset the code as the following:
```
export interface WorkspaceStart {
  currentWorkspaceId$: BehaviorSubject<string>;
  currentWorkspace$: BehaviorSubject<WorkspaceAttribute | null>;
  workspaceList$: BehaviorSubject<WorkspaceAttribute[]>;
  workspaceEnabled$: BehaviorSubject<boolean>;
}

export type WorkspaceSetup = WorkspaceStart;
```

Reply:
nice catch, this is not need now.

Reply:
I will keep `WorkspaceObservables` for now, should not be an issue, I think it will be needed in the future